test/e2e/logs_basic_v1alpha1_test.go

```diff
@@ -88,5 +89,14 @@ var _ = Describe(suite.ID(), Label(suite.LabelLogs), Ordered, func() {
 		It("Should have produced logs in the backend", Label(suite.LabelOperational), func() {
 			assert.LogsDelivered(proxyClient, loggen.DefaultName, backendExportURL)
 		})
+
+		Context("Metric instrumentation", Ordered, func() {
+			It("Ensures that telemetry_all_logpipelines metric is emitted", Label(suite.LabelOperational), func() {
+				assert.ManagerEmitsMetric(proxyClient,
+					Equal("telemetry_all_logpipelines"),
+					WithValue(BeNumerically(">", 0)),
```

Comment:
Equality is hard to test if there is a left-over LogPipeline from the previous execution, which happens from time to time. In this case, the best solution would be to execute it in an isolated job. But I don't think it's worth it since those 2 old metrics are not used.

Reply:
After a discussion with @rakesh-garimella and @a-thaler decided to drop old LogPipeline gauges completely since they are not used anywhere.